### ISSUE TITLE
Screenshots

### ISSUE THREAD

**Issue by 4513ECHO** [4513ECHO/vim-colors-hatsunemiku#1]
**Note: This colorscheme is still under development. Newer comments' screenshot is more similar with the actual situation.**

- [x] Markdown
- [x] terminal
- [x] Python
- [x] TypeScript
- [ ] light theme
- [ ] lightline
- [x] airline
- [ ] color palettes
- [x] filer
- [x] visual mode
- [x] completion popup
- [x] OGP image
- [ ] group-name
- [ ] vimdiff

**Comment by 4513ECHO** [comment]
terminal
![image](https://user-images.githubusercontent.com/81011153/158784172-5f95c6de-f8dd-46a6-b959-0acfd2fd1e3c.jpeg)

Python
![image](https://user-images.githubusercontent.com/81011153/158784261-739449a4-453f-43fd-a34d-b8e04556d1fe.jpeg)

**Comment by 4513ECHO** [comment]
Markdown
![image](https://user-images.githubusercontent.com/81011153/158782331-145b6628-3c88-4293-812d-f686e49d9d9c.jpeg)

**Comment by 4513ECHO** [comment]
completion popup (and TODO comment)
![image](https://user-images.githubusercontent.com/81011153/158976937-022814a7-2e15-48b9-9ae9-aac902158265.jpeg)
**Note:** I'm using [ddc.vim](https://github.com/Shougo/ddc.vim) for completion.

visual mode
![image](https://user-images.githubusercontent.com/81011153/158977077-62cacfaf-61e9-4cbc-83b2-d5384a0ed0e1.jpeg)

TypeScript (with [nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter))
![image](https://user-images.githubusercontent.com/81011153/158977130-2528ef12-4505-425d-85d6-e8c0438a0688.jpeg)

**Comment by 4513ECHO** [comment]
OGP image
![897994D0-3A93-495D-B67D-83660ACD64BE](https://user-images.githubusercontent.com/81011153/159166785-ad4a93f9-f7a7-41a2-9755-37807acb341c.jpeg)

**Comment by ghost** [comment]
**Airline**
![Airline_HatsuneMiku](https://user-images.githubusercontent.com/65461533/159806055-41fc31d1-ecfa-4dd5-a0fd-c60b8d32224e.png)

**Airline Light**
![Airline_HatsuneMiku_light](https://user-images.githubusercontent.com/65461533/159806091-4aeb94f4-a1c1-4dd2-ba0f-9fbb23f7ac17.png)

**Comment by 4513ECHO** [comment]
Thank you!

**Comment by ghost** [comment]
**Light Version** _(with NERDTree)_
![HatsuneMiku_Light](https://user-images.githubusercontent.com/65461533/159949066-f85c8330-4689-45a8-a2bf-bff342dc46c6.png)

**Comment by ghost** [comment]
**Palettes**
hatsunemiku:
![hatsunemiku](https://user-images.githubusercontent.com/65461533/163739293-ee2e6791-f5c4-49f8-9eef-b6770ec941e9.svg)

hatsunemiku_light:
![hatsunemiku_light](https://user-images.githubusercontent.com/65461533/163739297-fdc83876-bac1-4534-8dc9-da862f4a0bad.svg)

EDIT: Removed duplicate colours.
EDIT 2: Changed to SVG files.

**Comment by Kingorgg** [comment]
**airblade/vim-gutter:**
hatsunemiku:
![HatsuneMiku_Vim_Gutter](https://user-images.githubusercontent.com/65461533/163419403-b35703f9-452a-413a-977f-628e4679f732.png)

hatsunemiku_light:
![HatsuneMiku_Light_Vim_Gutter](https://user-images.githubusercontent.com/65461533/163419441-aa4f6d1e-07c2-47b3-b27c-3f5d27747b17.png)

**Comment by nyngwang** [comment]
Any update these days?

**Comment by 4513ECHO** [comment]
@nyngwang No, these days I have added no changes for this repository. However, the colorscheme is already stable and you can use it. It is welcome to upload the screenshot to this issue.
